### ISSUE TITLE
Bug: Hover tooltips are not readable via screen reader.

### ISSUE THREAD

**Issue by ShrimpCryptid** [allen-cell-animated/nucmorph-colorizer#309]
## Description

- Tooltips are stored outside of the normal document flow, and as a result can't be read easily by screen readers.

## Expected Behavior

- Update all locations where tooltip text is used to provide additional context or information, and add a `VisuallyHidden` text component with the same information.
- Optional thought: Make an accessible tooltip with default properties (hover + focus enabled) that automatically injects this?

ex:

![Image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/cb364da2-5e15-467c-87d7-21647343cc96)

![Image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/b8093b35-a127-4fb0-8815-6de3b90dc8a8)

![Image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/69ec61a1-8fc0-4ed5-ba5e-1f26af5d54bf)

## Reproduction

_A minimal example that exhibits the behavior._

## Environment

_Any additional information about your environment. E.g. OS version, python version, browser etc._
